static/scss/partials/glocal-menu.scss

```diff
@@ -144,10 +144,14 @@ glocal-menu {
     padding: $spacing-md var(--glocalPaddingUnit);
     width: 100%;
     background-color: rgba(255, 255, 255, 0);
+    color: $color-violet-90;
+    font-size: 14px;
 
     img {
-        padding: 0 $spacing-xs;
-        padding-right: $spacing-md;
+        display: inline-block;
+        width: var(--glocalSmallAvatarDimensions);
+        margin-right: 12px;
+        opacity: .9;
```

Comment:
This is copied from https://github.com/mozilla/fx-private-relay/pull/1199/files#diff-9085c794e76019c70b362951988c750e222e4c5c79595d12eb01f8870f9e8de2R177-R185